plugins/org.eclipse.che.editor.jupyter/1.0.0/meta.yaml

```diff
@@ -6,3 +6,12 @@ title: Jupyter Notebook as Editor for Eclipse Che
 description: Jupyter Notebook as Editor for Eclipse Che
 icon: https://jupyter.org/assets/main-logo.svg
 url: https://github.com/ws-skeleton/che-editor-jupyter/releases/download/untagged-cc4e4c9a7741551b6776/che-editor-plugin.tar.gz
+publisher: Red Hat, Inc.
+category: Editor
+repository: https://github.com/ws-skeleton/che-editor-jupyter/
+firstPublicationDate: Jan 11, 2019
```

Comment:
what is the format of this date ?

Reply:
We have no inputs about that. What you can suggest?

Reply:
https://en.wikipedia.org/wiki/ISO_8601 ?

Reply:
ok, I'll use this format